# ros configuration file

/**:
  ros__parameters:
    odometry_topic: "none"
    pose_topic: "self_localization/pose"
    detections_topic: "processed_gate_poses_array"
    corrected_localization_topic: "slam/corrected_localization"
    map_frame: "drone0/map"
    odom_frame: "drone0/odom"
    robot_frame: "drone0/base_link"
    viz_main_markers_topic: "slam_viz/markers/main"
    viz_temp_markers_topic: "slam_viz/markers/temp"
    main_graph_odometry_distance_threshold: 2.0
    main_graph_odometry_orientation_threshold: 1.0
    temp_graph_odometry_distance_threshold: 0.2
    temp_graph_odometry_orientation_threshold: 1.0
    odometry_is_relative: False
    generate_odom_map_transform: False
    fixed_objects:
      gate_1:
        type: "aruco"
        pose: [5.6, 1.5, 1.37, 0.0]
      gate_2:
        type: "aruco"
        pose: [14.3, -1.06, 1.37, 0.0]
